addons/osio-addon/src/main/java/io/fabric8/launcher/osio/projectiles/OsioProjectileContext.java

```diff
@@ -15,11 +15,11 @@
 public class OsioProjectileContext extends OsioImportProjectileContext implements LauncherProjectileContext {
 
     @FormParam("missionId")
-    @NotNull(message = "Mission is required")
+    @NotNull(message = "missionId is required")
```

Comment:
`missionId`?

Reply:
I got those error while developing and it was really confusing to have a required message but not the field name it self.

Reply:
But the field name is `mission`, not `missionId`?

Reply:
@FormParam("runtimeId")

Reply:
we need to rename the field I think then it is confusing either way

Reply:
Yeah, I changed the `/launcher/zip` parameters but forgot to change this one, let's do it in another PR